daffodil-io/src/main/scala/org/apache/daffodil/io/DirectOrBufferedDataOutputStream.scala

```diff
@@ -33,81 +34,82 @@ import org.apache.daffodil.util.Maybe
 import org.apache.daffodil.util.Maybe._
 import org.apache.daffodil.util.MaybeULong
 import org.apache.daffodil.util.Misc
+import org.apache.daffodil.util.Logging
 import java.io.File
 
 /**
  * This simple extension just gives us a public method for access to the underlying byte array.
  * That way we don't have to make a copy just to access the bytes.
  */
-private[io] class ByteArrayOrFileOutputStreamWithGetBuf(
-  chunkSizeInBytes: Long = 4096,
-  maxBufferSizeInBytes: Long = 2000 * (1 << 20))
-  extends java.io.ByteArrayOutputStream {
-  private var _byteArrayOS = this
-  private var _fileOS: java.io.FileOutputStream = null
-
-  var isFile: Boolean = false
+private[io] class ByteArrayOutputStreamWithGetBuf() extends java.io.ByteArrayOutputStream {
+  def getCount = count
```

Comment:
I don't think this getCount is used anymore? I think ByteArrayOrFileOutputStream keeps track of the count regardless of if the stream is array based or file based?

Reply:
You are correct. This will be removed.